integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/smoke/YarnSmokeTest.java

```diff
@@ -31,16 +34,16 @@
     private static final String INSTANCE_GROUP_ID = "ig";
 
     private static final String POST_CLUSTER_INSTALL_RECIPE_CONTENT = Base64
```

Comment:
There is already a directory for recipes used in integration test. It would be more readable to have these as separate files, and the the should read the content from those files.
./integration-test/src/main/resources/recipes/post-install.sh